src/app/components/AboutPage.jsx

```diff
@@ -0,0 +1,97 @@
+import Image from "next/image";
+import { FaFacebookF, FaXTwitter, FaLinkedinIn } from "react-icons/fa6";
+
+const Button = ({ text, className }) => (
+  <button
+    className={`bg-gray-800 text-white px-4 py-2 rounded hover:bg-gray-700 ${className}`}
+  >
+    {text}
+  </button>
+);
+
+const Section = ({ title, children }) => (
+  <div className="border border-about-bg p-6 rounded-lg space-y-4">
+    <div className="flex items-center space-x-2">
+      <span className="text-blue-400">✦</span>
+      <h2 className="text-xl font-semibold">{title}</h2>
+    </div>
+    {children}
+  </div>
+);
+
+const SocialMediaButtons = () => (
+  <div className="inline-flex border border-about-bg px-4 py-2 rounded-3xl mr-6	 ">
+    <a
+      href="https://facebook.com"
+      target="_blank"
+      rel="noopener noreferrer"
+      className="text-gray-500 hover:text-white mr-2"
+    >
+      <FaFacebookF />
+    </a>
```

Comment:
I would refactor it to a separate component as you have to copy it three times
Something like SocialMediaLink and just use it like this:
<SocialLink href="https://facebook.com">
      <FaFacebookF />
</SocialLink>